SSA/Experimental/IntrinsicAsymptotics.lean

```diff
@@ -1148,7 +1176,7 @@ def p1 : PeepholeRewrite ExOp [.nat, .nat] .nat:=
   { lhs := m, rhs := r, correct :=
     by
       rw [m, r]
-      simp_peephole
+      simp_peephole [add, cst]
```

Comment:
I would prefer to see an alias for `simp_peephole [add, cst]` to reduce duplication, but would not consider this a blocker

Reply:
I prefer the current, explicit version.

Reply:
I have no idea what [add, cst] means. Why do we now need to add arguments and before simp_peephole was generic?

Reply:
`simp_peephole` was never generic, we needed to unfold the (specific) definitions of `add` and `cst` from the example dialect. From now on, since we have multiple dialects, one will need to write `simp_peephole [op1, op2, ..., opn]` for the ops of a dialect

Reply:
We can also create an alias as @alexkeizer suggests, one for each dialect. Fundamentally, there's going to be a version of `simp_peephole` for each dialect, as we need to tell `simp` to unfold ops.

Reply:
I see l
Thank you.